server/src/test/java/io/unitycatalog/server/base/model/BaseModelCRUDTest.java

```diff
@@ -248,6 +266,8 @@ public void testModelCRUD() throws ApiException {
     assertThat(updatedModelVersionInfo.getStatus())
         .isEqualTo(ModelVersionStatus.PENDING_REGISTRATION);
     assertThat(updatedModelVersionInfo.getVersion()).isEqualTo(3L);
+    assertThat(updatedModelVersionInfo.getStorageLocation())
```

Comment:
If we are allowed the update model names, can we also do a check that despite updating the model name, the storage location stays the same.

Reply:
Yep.  Can add that.  Ids and storage locations (since they are now purely ID based) should remain consistent across updates.

Reply:
Oh actually....thats what this check is doing.  I compare the updatedMV against the originally pulled MV

Reply:
I meant change the name of the registered model. For example create a model , create a model version. Update the  registered model name. And make sure the model version is reachable from the updated model. We can add those tests later.